portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/ManagementToolbar.testcase

```diff
@@ -103,6 +103,45 @@ definition {
 		}
 	}
 
+	@description = "LPS-147850. Assert the order button can sort results"
+	@priority = "5"
+	test OrderButtonCanSortOptions {
+		task ("When the user opens any section with the management toolbar") {
+
+			// Using tags as workaround until LPS-150056
+
+			JSONAssettag.addTag(
+				groupName = "Guest",
+				tagName = "a");
+
+			JSONAssettag.addTag(
+				groupName = "Guest",
+				tagName = "b");
+
+			Tag.openTagsAdmin(siteURLKey = "guest");
+		}
+
+		task ("And clicks on the order button") {
+			Click(
+				key_filter = "Order",
+				locator1 = "ManagementBar#FILTER_OR_ORDER",
+				value1 = "Order");
+		}
+
+		task ("Then the user can select one item to order the results") {
+			AssertClick(
+				key_item = "Descending",
+				locator1 = "ManagementBar#ACTIVE_DROPDOWN_ITEM",
+				value1 = "Descending");
+		}
+
+		task ("And the results are displayed accordingly to the selected option") {
+			AssertElementPresent(locator1 = "xpath=(//*[@data-qa-id='row'][1]//*[contains(.,'b')])");
+
+			AssertElementPresent(locator1 = "xpath=(//*[@data-qa-id='row'][2]//*[contains(.,'a')])");
```

Comment:
I didn't the paths to a file if I'm going to be using clay sample portlet later. Thoughts? :thinking:

Reply:
This is fine with me.